sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/SearchIndexAsyncClient.java

```diff
@@ -722,6 +727,21 @@ SearchClientBuilder getSearchClientBuilder(String indexName) {
         return withContext(context -> deleteSynonymMapWithResponse(synonymMap.getName(), etag, context));
     }
 
+    /**
+     * Helper method to build list of {@link SearchField}.
+     *
+     * @param model The model class where {@link SearchField} converts from.
+     * @param options The option property bag.
+     * @return The list {@link SearchField} for search index schema.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public static Mono<List<SearchField>> buildSearchField(Class<?> model, FieldBuilderOptions options) {
+        if (options == null) {
+            return Mono.just(FieldBuilder.build(model, new JacksonJsonSerializerProvider().createInstance()));
```

Comment:
This should use `JsonSerializerProviders.createInstance()`, if `azure-core-serializer-json-jackson` is excluded this will fail to compile.

Reply:
Sure. Will change to MemberNameConverterProviders.createInstance()